Sources/StreamChatUI/CommonViews/AvatarView/ChatChannelAvatarView.swift

```diff
@@ -30,31 +47,293 @@ open class ChatChannelAvatarView: _View, ThemeProvider, SwiftUIRepresentable {
                 preferredSize: .avatarThumbnailSize,
                 components: components
             )
+            presenceAvatarView.isOnlineIndicatorVisible = false
             return
         }
 
-        let (avatarURL, isOnlineIndicatorVisible): (URL?, Bool) = {
-            // Try to get the explicit channel avatar first
-            if let avatarURL = channel.imageURL {
-                return (avatarURL, false)
-            }
-
-            // TODO: https://stream-io.atlassian.net/browse/CIS-652
-            // this is just placeholder implementation:
-
-            let firstOtherMember = channel.lastActiveMembers
-                .sorted { $0.memberCreatedAt < $1.memberCreatedAt }
-                .first(where: { $0.id != content.currentUserId })
-
-            return (firstOtherMember?.imageURL, firstOtherMember?.isOnline ?? false)
-        }()
-
+        loadAvatar(for: channel)
+    }
+    
+    open func loadAvatar(for channel: ChatChannel) {
```

Comment:
question just to double-check, we don't show any loading state, we showing some hardcoded image instead?